example-app/SantokuApp/src/features/qa-home/pages/HomePage.tsx

```diff
@@ -180,26 +181,38 @@ export const HomePage: React.FC<HomePageProps> = ({
               </Text>
             </Box>
             {!isEventsLoading && <EventList data={events} />}
-            <StyledRow px="p24" py="p32" justifyContent="space-between" alignItems="center">
-              <Text variant="font20Bold" lineHeight={24} letterSpacing={0.18}>
-                {m('質問')}
-              </Text>
-              <StyledRow space="p32" alignItems="center">
-                <StyledTouchableOpacity onPress={setVisibleSortSheet}>
-                  <SortIllustration color={sortIconColor} />
-                </StyledTouchableOpacity>
-                <StyledTouchableOpacity onPress={showUnderDevelopment}>
-                  <FilterAltIllustration />
-                </StyledTouchableOpacity>
-                <StyledTouchableOpacity onPress={setVisibleTagSheet}>
-                  <LocalOfferIllustration color={tagIconColor} />
-                </StyledTouchableOpacity>
-              </StyledRow>
-            </StyledRow>
           </>
         }
         data={questionItems}
-        renderItem={QuestionListCard}
+        renderItem={listRenderItemInfo => {
+          if (listRenderItemInfo.item === questionItems[0]) {
```

Comment:
シンプルに、indexの比較で良いと思いました。
```suggestion
          if (listRenderItemInfo.index === 0) {
```

Reply:
FYI: markdown で リンクを作らなくても、 hash id や URL をそのまま書いておけば GitHub がいい感じにしてくれます。

- 4b799111845f5557c9146716ad101afe749d6403
  - `4b799111845f5557c9146716ad101afe749d6403`
- https://github.com/ws-4020/mobile-app-crib-notes/pull/1055/commits/4b799111845f5557c9146716ad101afe749d6403
  - `https://github.com/ws-4020/mobile-app-crib-notes/pull/1055/commits/4b799111845f5557c9146716ad101afe749d6403`
- 4b79911
  - `4b79911`
  - 短くても行けるが衝突の可能性はある（衝突したらどうなるかは未確認）